packages/x-date-pickers/src/DateCalendar/DayCalendar.tsx

```diff
@@ -110,6 +110,7 @@ export interface DayCalendarProps<TDate>
 const useUtilityClasses = (ownerState: DayCalendarProps<any>) => {
   const { classes } = ownerState;
   const slots = {
+    root: ['root'],
```

Comment:
I might be missing some context here, but is there a specific reason why we did not have this before? 🤔 I use the root of each subcomponent of the pickers for the highlighting in the demo and I needed this to be able to highlight the entire `DayCalendar`

Reply:
Probably just forgotten about it. 🙈 
But if we are adding it in, it has to be done properly and all the necessary changes should also be done on `dayCalendarClasses` to expose this slot for extension. 😉

Reply:
I created the components (at least the date one) and I definitely forgot it :laughing: